ws/sha1.ts

```diff
@@ -370,13 +382,12 @@ export class Sha1 {
   arrayBuffer() {
     this.finalize();
 
-    let buffer = new ArrayBuffer(20);
-    let dataView = new DataView(buffer);
-    dataView.setUint32(0, this.h0);
-    dataView.setUint32(4, this.h1);
-    dataView.setUint32(8, this.h2);
-    dataView.setUint32(12, this.h3);
-    dataView.setUint32(16, this.h4);
-    return buffer;
+    let array = new Uint32Array(5);
```

Comment:
no reassignment, why are we not using `const` (previous code should have been `const` too, so yeah, asking you to mop up other people's code... 😄 Why can't we do `Uint32Array.of()` here?

Reply:
I thought about at first time, but `Uint32Array.of()` little slower than current apprach. But I could switch to `Uint32Array.of` of course. This should looks cleaner.